Extensions/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageLocalization/Module.jsx

```diff
@@ -35,9 +35,10 @@ class Module extends Component {
 
         if (module.Exist) {
             const className = "module-row" + (module.IsDeleted ? " deleted" : "");
+            const toolTip = ["Shared: " + module.IsShared, module.ModuleInfoHelp].join("<br>");

```

Comment:
@mikebigun Shouldn't this string be localized?

Reply:
@daguiler you are right. It's done.

Reply:
Sorry to insist @mikebigun, but localization should also be applied to `module.IsShared`

Reply:
@daguiler pls, check now.